src/components/fetcher/index.js

```diff
@@ -39,15 +39,21 @@ const Fetcher = ({
 
         let fetchErrorCounter = 0;
         let emptyDataCounter = 0;
-
+        console.log("FUNCTION CALL +++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
         data && data.map(async ({ root, sidekey, time = null }) => {
           try {
-            const mode = sidekey ? Mode.Old : Mode.Public;
-            const reader = new Reader(ctx, client, mode, root, sidekey || '9'.repeat(81));
-            const message = await reader.next();
-            const payload = get(message, 'value[0].message.payload');
-            if (payload) {
-                saveData(JSON.parse(trytesToAscii(payload)), time);
+
+            const message  = await mamFetch('https://api.lb-0.testnet.chrysalis2.com', root, 'restricted', sidekey)
```

Comment:
Please add the provider URL into config file or get it from the cloud config call

Reply:
added to config file